R/api_call.R

```diff
@@ -0,0 +1,87 @@
+#' @importFrom httr status_code POST
```

Comment:
You should add a copyright header here (Apache 2), and probably put yourself and Sonatype as authors, as you've significantly modified the code from what I can tell!

Reply:
- Added me to the author list. 

- Typically in R packages, the DESCRIPTION contains the licence/copyright and so the header isn't required. Your package has the Apache 2 in the DESCRIPTION, so is it needed for the files? Proviso: I know nothing about licences